addons/base_setup/models/res_config_settings.py

```diff
@@ -115,3 +117,9 @@ def _compute_company_informations(self):
 
         for record in self:
             record.company_informations = informations
+
+    @api.model
+    def _install_modules(self, modules):
+        if not (getattr(threading.currentThread(), 'testing', False) or self.env.registry.in_test_mode()) and 'google_drive' in dict(modules).keys():
+            raise UserError(_('The Google Drive integration has been deprecated due to changes in Google authentation APIs.'))
+        return super()._install_modules(modules)
```

Comment:
Let's not.

Doesn't switching drive's `installable` key to `False` suffice? Why is the test mode even involved?

Reply:
@xmo-odoo According to @kba-odoo : ``I couldn't set installable: False as it will prevent loading the module on existing db when the server restarts``

Reply:
if too shitty, we can leave the module as installable and just prevent it from working properly. not ideal, for sure...